stubs/tifftools/constants.pyi

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Generator, Generic, TypeVar, overload
+
+_ConstantAttr = Any
+
+class TiffConstant(int):
+    value: int
+    name: str
+    def __init__(self, value: int, constantDict: dict[str, _ConstantAttr]) -> None: ...
+    def __getitem__(self, key: str) -> _ConstantAttr: ...
+    def get(self, key: str, default: _ConstantAttr = ...) -> _ConstantAttr: ...
+
+_ConstantT = TypeVar('_ConstantT', bound=TiffConstant)
```

Comment:
My understanding is that this line creates a new type called `_ConstantT` which represents `TiffConstant` and any subtype of `TiffConstant`. Is that correct?

In that case, I think the name `_TiffConstantT` is more clear.

Reply:
Agreed. I'll change the name.